print_list.js

```diff
@@ -0,0 +1,34 @@
+
+class Node {
+  constructor(value, next) {
+    this.value = value;
+    this.next = next;
+  }
+
+  getNext() {
+    return this.next;
+  }
+
+  setNext(next) {
+    this.next = next;
+  }
+
+  getValue() {
+    return this.value;
+  }
+
+  setValue(value) {
+    this.value = value;
+  }
```

Comment:
```suggestion
attr_accessor :value, :next
```
This can be placed at the top of the file under `class Node` to automatically generate getters and setters for your member variables